executor/runtime/docker/docker_linux.go

```diff
@@ -44,6 +43,7 @@ const (
 	systemServiceStartTimeout = 90 * time.Second
 	umountNoFollow            = 0x8
 	sysFsCgroup               = "/sys/fs/cgroup"
```

Comment:
Are you sure this wouldn't all be easier with the `--cgroup-parent` option on the docker deamon?

Reply:
I think that changes the parent cgroup under `/sys/fs/cgroup/systemd/`? Our current requirement is to move the pid under the `/sys/fs/cgroup/unified/` path to enable cgroupv2 ebpf programs.

Reply:
So it wouldn't be `--cgroup-parent=/sys/fs/cgroup/unified/` ?

Reply:
I believe `--cgroup-parent` controls the parent directory under `/sys/fs/cgroup/systemd` for cgroups controlled by systemd. i.e.
`/sys/fs/cgroup/systemd/containers.slice/titus-executor@default__3059b7ee-63ac-43ff-adf6-c0cf7d399a85.service/8f5534ff716e9492999b5a859cdc5342eb791e9f80d6e35eac42c81eb0c90cd2/`.

The code above moves the pid to the unified hierarchy under `/sys/fs/cgroup/unified`